recipes/fastscapelib-f2py/build.sh

```diff
@@ -1 +1,6 @@
-$PYTHON -m pip install . --no-build-isolation --no-deps --ignore-installed --no-cache-dir -vvv
+if [ $(uname) == Darwin ]; then
+    # scikit-build doesn't support setting MACOSX_DEPLOYMENT_TARGET
+    $PYTHON -m pip install . --global-option="build_ext" --global-option="--plat-name=macosx-10.9-x86_64" --no-build-isolation --no-deps --ignore-installed --no-cache-dir -vvv
```

Comment:
Use `$MACOSX_DEPLOYMENT_TARGET` instead of `10.9`

Reply:
> do you know if there is a way to pass cmake options like [...] to scikit-build via the pip or python setup.py command-line

Any option can be passed to cmake when using `setup.py`, the usage is the following:

```
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...] [skbuild_opts] [-- [cmake_opts] [-- [build_tool_opts]]]
or: setup.py --help [cmd1 cmd2 ...]
or: setup.py --help-commands
or: setup.py cmd --help
```
See https://scikit-build.readthedocs.io/en/latest/usage.html#command-line-options

This means you could do something like this:

```
python setup.py bdist_wheel -- -DCMAKE_Fortran_COMPILER:FILEPATH=/path/to/compiler -DF2PY_EXECUTABLE:FILEPATH=/path/to/f2py
```

Reply:
Thanks @jcfr, good to know! I should have looked at the docs more in depth before asking.

Reply:
> I should have looked at the docs more in depth before asking.

I created an issue so that we better document the passing of CMake option from setuptools and pip.

Reply:
With the upcoming version of scikit-build, it will be possible to pass CMake option starting with `-D` directly from pip.

See https://github.com/scikit-build/scikit-build/pull/427